mitreattack/navlayers/README.md

```diff
@@ -30,6 +30,10 @@ This folder contains modules and scripts for working with ATT&CK Navigator layer
 | [matrix_gen](https://github.com/mitre-attack/mitreattack-python/blob/master/mitreattack/navlayers/exporters/matrix_gen.py) | Provides a means by which to generate a matrix from raw data, either from the ATT&CK TAXII server or from a local STIX Bundle. |
 | [svg_templates](https://github.com/mitre-attack/mitreattack-python/blob/master/mitreattack/navlayers/exporters/svg_templates.py) | Provides a means by which to convert a layer file into a marked up svg file. |
 | [svg_objects](https://github.com/mitre-attack/mitreattack-python/blob/master/mitreattack/navlayers/exporters/svg_objects.py) | Provides raw templates and supporting functionality for generating svg objects. |
+##### Collections Scripts
+| script | description |
+|:-------|:------------|
+|[col_to_md](https://github.com/mitre-attack/mitreattack-python/blob/master/mitreattack/navlayers/collections/col_to_md.py)| Provides a means by which to convert a collection index into a human-readable markdown file. More information can be found in the corresponding [section](#col_to_md.py) below.|
```

Comment:
As mentioned in [this comment](https://github.com/mitre-attack/mitreattack-python/issues/29#issuecomment-900326058), collection features should be in their own submodule, not a part of `navlayers`. In other words they should be in an organizational grouping at the same level as navlayers and attackToExcel.

Reply:
My bad - I misunderstood that line. I'll move it, and adjust the documentation accordingly.